bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -75,11 +75,11 @@
 	</channel-type>
 
 	<channel-type id="fanspeed" advanced="true">
-		<item-type>Number:Frequency</item-type>
+		<item-type unitHint="rpm">Number:Frequency</item-type>
 		<label>Fan Speed</label>
 		<description>Actual measured rotation speed of the fan</description>
 		<category>Fan</category>
-		<state readOnly="true" pattern="%d %unit%"/>
+		<state readOnly="true" pattern="%d rpm"/>
```

Comment:
This should be reverted to `%unit%` also.

Reply:
Why ? My proposal is to display in rpm by default whatever the internal unit of the item (Hz by default).

Reply:
It's not the way it is usually done. The unit provided by the API is rpm, you provide a unitHint in that way also, so default state pattern should not be changed.

Reply:
In fact, since the unit was added on item, the proper way is to not use anymore `%unit%` in channel state pattern. When we define a state pattern on a channel type as a developer, it is because we think this pattern with the proposed formatting and unit is the most appropriate as default for the display of this channel, whatever the choice of unit done for the item.
This is even more important when the default unit is not very appropriate, like Hz in the current case.

Reply:
Understood, you know better than me the orthodox way. I approve.

Reply:
@J-N-K @mherwege @openhab/add-ons-maintainers : please confirm that my answer to @clinique is correct.

Reply:
I confirm that the channel state description pattern would only be needed if the developer thinks there is a reason to show a specific pattern, irrespective of the item unit.
Note it used to be good to set %unit% because there was no default state description pattern for an item. But currently the default for the item would set %unit% anyway.